charts/victoria-logs-single/files/dashboards/generated/victorialogs.yaml

```diff
@@ -840,12 +840,91 @@ panels:
     refId: A
   title: Log stream churn rate
   type: timeseries
+- datasource:
```

Comment:
I can't find this dashboard in master branch, any chance this is a part of another PR commited here by accident?

Reply:
all dashboards were generated by a sync script

Reply:
Sorry for the confusion, looks like Goland did not bother with updating files on disk until "hard refresh" was triggered manually. LGTM